app/code/Magento/InventoryInStorePickupFrontend/view/frontend/web/js/view/form/element/email.js

```diff
@@ -0,0 +1,22 @@
+/**
```

Comment:
No reason to create a custom component. You could provide template and links configuration in checkout_index_index.